src/dev/eslint/run_eslint_with_types.ts

```diff
@@ -85,6 +85,7 @@ export function runEslintWithTypes() {
                 ),
                 ...(project.config.exclude ?? []).flatMap((p) => ['--ignore-pattern', p]),
                 ...['--ignore-pattern', '**/*.json'],
+                ...['--ignore-pattern', '**/*.d.ts'],
```

Comment:
Otherwise the parser picks up d.ts files from typings and then complains because they don't belong to any known projects.